scripts/calibrate_from_rotation_series.py

```diff
@@ -546,7 +540,6 @@ def parse_reflection_tables(cfg, instr, grain_id, refit_idx=None):
 # for det_key, panel in instr.detectors.iteritems():
 #     hkls_refit = hkls[det_key][idx_new[det_key], :]
 #     xyo_det_refit = xyo_det[det_key][idx_0[det_key], :]
-#     pass
```

Comment:
All of this commented out code should probably be removed, rather than just the pass.

Reply:
I don't want to delete this in this PR.  Maybe people use this sometimes by uncommenting it.  Could be deleted in a different PR